wpeverest/everest-forms
# Fix - Field option sections get freezed

### All Submissions:






### Changes proposed in this Pull Request:
If you drop two fields in such manner: drop a second field while the first field is still loading, field option sections for the first field gets freezed and you can't toggle any of the sections. This PR resolves the issue.

### How to test the changes in this Pull Request:

1. Create a new form
2. Drop a field
3. Drop another field while that field is still loading
4. Now go to the Field Options tab of the first field
5. Try to toggle the option sections. It should now work properly.

### Types of changes:




### Other information:




### Changelog entry

> Fix - When you drop a second field while the first one is still loading, `Field Options` get freezed for the first field
